PowershellTools.Common/ServiceManagement/DebuggingContract/PowershellBreakpoint.cs

```diff
@@ -25,5 +25,12 @@ public PowershellBreakpoint(string file, int line, int column)
             Line = line;
             Column = column;
         }
+
+        public bool Equals(PowershellBreakpoint other)

```

Comment:
If you implement IEquatable<T>, you should also override the base implementations of Object.Equals(Object) and GetHashCode